robocop/config/config.py

```diff
@@ -23,8 +23,8 @@ def __init__(self):
         self.configure = list()
         self.format = "{source}:{line}:{col} [{severity}] {msg_id} {desc}"
         self.paths = []
-        self.include_patterns = None
-        self.exclude_patterns = None
+        self.include_patterns = list()
```

Comment:
Why not to just create empty list by calling `self.include_patterns = []`? I think it's more concise and even a bit faster.